Assignment.py

```diff
@@ -0,0 +1,1604 @@
+a=input()
+count=0
+for i in a:
+    if (i>="A" and i<="Z"):
+        count=count+1
+if count>=1:
+    print("yes the string contains capital letter")
+else:
+    print("the string does not contain capital letter")
+
+
+# In[9]:
+
+
+#40take a string from the user and check contains at least one small letter or not?
+a=input()
+count=0
+for i in a:
+    if (i>="a" and i<="z"):
+        count=count+1
+if count>=1:
+    print("yes the string contains small letter")
+else:
+    print("the string does not contain small letter")
+
+
+# In[18]:
+
+
+#41  Print the first 100 odd numbers 
+result=""
+for i in range(1,200):
+    if ((i%2)!=0):
+        result=result+str(i)+" "
+print(result)        
+
+
+# In[17]:
+
+
+#42 Determine the factors of a number entered  by the user 
+n=int(input())
+for i in range(1,n+1):
+    if n%i==0:
+        print(i)
+
+
+# In[35]:
+
+
+#43 Play a number guessing game (User enters a guess, you print YES or Higher or Lower).
+#This should continue until and until user gives a correct number or want to quit in the middle. 
+import random
+a=random.randint(1,100)
+while True:
+    n=(input("enter number\n"))
+    if n=="q":
+        print("the correct answer is:"+str(a))
+        break
+    elif n==str(a):
+        print("you have guess a correct number")
+    elif n<str(a):
+        print("please enter higher number")
+    elif n>str(a):
+        print("please enter lower number")
+
+
+# In[2]:
+
+
+#44.Take two numbers from the user a,b check whether a is divisible by b or not? 
+a=int(input())
+b=int(input())
+if a%b==0:
+    print("a is divisible by b")
+else:
+    print("a is not divisible by b")
+    
+
+
+# In[3]:
+
+
+#45.Find the sum of all the multiples of 3 or 5 below 1000 
+sum=0
+for i in range(1,1000):
+    if (i%3==0) or (i%5==0):
+        sum=sum+i
+print(sum)        
+
+
+# In[4]:
+
+
+#46.Write a program to find out big of two numbers 
+a=int(input())
+b=int(input())
+if a>b:
+    print(a)
+else:
+    print(b)
+
+
+# In[6]:
+
+
+#47. Write a program to find out biggest number in the given numbers. 
+n=list(map(int,input().split()))
+print(max(n))
+
+
+# In[54]:
+
+
+#48. find out the index of  third occurrence of given substring (apple apple orange guava apple,apple,output 5)
+sentence=input().split()
+substring=input()
+count=0
+for i in range(len(sentence)):
+    if sentence[i]==substring:
+        count=count+1
+        if count==3:
+            print(i)
+            break
+
+
+# In[23]:
+
+
+#49.find out the index nth occurrence of given substring 
+sentence=input().split()
+substring=input()
+count=0
+for i in range(len(sentence)):
+    if sentence[i]==substring:
+        count=count+1
+        result=i
+print(result)        
+
+
+# In[28]:
+
+
+#50.Take some single digit numbers from the user and findout min, maximum, sum, average 
+n=list(map(int,input().split()))
+print(min(n))
+print(max(n))
+sum=0
+for i in n:
+    sum=sum+i
+print(sum)
+print(sum/len(n))
```

Comment:
your code is right but you have to explain what is the answer like
maximum value is=5
minimum value is=2
like that 
please check again